airqo-platform/AirQo-frontend
# [NetManager] Fix Pending Issues 691

#### Summary of Changes (What does this PR do?)
- Fixes device maintenance logs (device Overview, adding, updating, deleting a maintenance log)
- Enable adding devices and editing device name(long_name)
- Enable adding site
- Enable deleting a site
- Enable adding user
- Fix device management network chart
- Fix device management chat `last updated` format
- Sort device leaderboard by `uptime` and `deviceName`
- Merge device overview chart components inorder to share `status` state
- Handle redux empty state update

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Install requirements `npm install`
* Start netmanager application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* Ensure the issues stated in the `Summary of Changes` are fixed

#### What are the relevant tickets?
- [PLAT-691](https://airqoteam.atlassian.net/browse/PLAT-691)

#### Screenshots (optional)
